bootstrap/main.go

```diff
@@ -54,20 +53,8 @@ func main() {
 		die("There was an error fetching the current cluster info: %s\n", err)
 	}
 
-	fmt.Printf("\nThis will install Weave Cloud on the following cluster:\n")
-	fmt.Printf("    Name: %s\n    Server: %s\n\n", cluster.Name, cluster.ServerAddress)
-	fmt.Printf("Please run 'kubectl config use-context' or pass '--kubeconfig' if you would like to change this.\n\n")
-
-	confirmed, err := askForConfirmation("Would you like to continue?", opts.AssumeYes)
-	if err != nil {
-		die("There was an error: %s\n", err)
-	}
-	if !confirmed {
-		fmt.Println("Cancelled.")
-		return
-	}
+	fmt.Printf("Installing Weave Cloud agents on %s at %s", cluster.Name, cluster.ServerAddress)
```

Comment:
why is the address useful?

Reply:
the names are not always very descriptive :(

Reply:
but the address is?!?

Reply:
ok. fair enough.

Reply:
We need to make up our mind whether it's "agent" or "agents". Further down we have a message that says "agent".

Reply:
In this case it feels like "Weave Cloud agents" is the correct message to show output to the user because the single update agent is transparent to the user.